.github/workflows/chrome_v112.yml

```
@@ -0,0 +1,20 @@
+name: chrome_v112
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          cache: "yarn"
+      - uses: browser-actions/setup-chromium@v1
+        with:
+          chrome-version: "112.0"
```

The GitHub action attempts to retrieve from:
```console
Acquiring 112.0 from https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F112.0%2Fchrome-linux.zip?alt=media
```

When exploring the [Linux_x64 folder](https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Linux_x64/), the version is `112001`

```suggestion
          chrome-version: "112001"
```

---

The numbers in Linux_x64 folder are commit positions, and not version numbers